samples/NativeCarter/CarterNativeSample.csproj

```diff
@@ -6,6 +6,8 @@
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="Microsoft.AspNetCore" Version="2.0.0" />
+    <PackageReference Include="Microsoft.AspNetCore.Mvc.Core" Version="2.0.1" /> 
```

Comment:
Any ideas why this fixes the problem?  I know its a sample app but Carter apps shouldn't depend on MVC